### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by VirtualLusamine** [OpenVoiceOS/ovos-buildroot#164] [Q&A] [answered]
Ive wanted to have a Private AI assistant for a while but with mycroft and others they never had a discord channel that helped attract other members, discord could be a amazing place to discuss the project, get support, suggest changes, make forums, and so many people use it and its really the standard for any community now these days so id request making a discord server to acompany the development of this project.

I can help with questions about discord or even volunteer to help make or moderate it. I just want it to be more visible to more people so it doesnt die like mycroft or be inconvenient to access by people that use Discord and are not sure if they should trust Matrix as to me it isnt a proven company.

**Comment by ChanceNCounter** [comment]
Matrix isn’t a company at all. It’s like email.

In my other life, I am a game developer. Hence, I’m going to finish answering this question like you had asked it on Discord.

—-

You see the fkn unprofessional tone I dropped into here? The only people on Discord are gamers, game developers, and teenagers.

Our chat rooms aren’t IMs and not a single person has ever discovered us through the chat

Meanwhile, Discord *is* a company, and a service, and if we switched to it we would be stuck with it. The worst thing that can happen to Matrix is we might have to change providers

As for the rest of it, this sentence will demonstrate to my colleagues why we don’t wanna be within 500 miles of Discord:

One of the game dev Discord servers I hang out in has to ban a kid at least once a week for being too young to legally internet. No.

**Reply by VirtualLusamine** [reply]
I understand your concerns. i really do. But you have a very wild view of what discord is. Its about building a community. If you make a server for a game you are gonna have teens that have behavioral issues. If you make a server for meth you will have the averege NY citizen joining. If you make something that is privacy focused, based on linux for linux that is heavily tied to the open source community.  You can expect to attract those people. And even if so and you dont want to risk getting bad people in. You can just, only have or give the discord server to the people in the forums or that you know would be good for the community.
Sending what is basically a email as a messege every time you wanna talk (assuming how it works by what you said feel free to correct me) is not very natural as you might asume, it makes progress slow and not fluid. It makes expression of ideas slow and the regection of stupid ones slow as well.
I find it extremely hard to engage with a community thats is slow and relies on those methods, and so do many other people.
Im not a kid, im a 18 y/o woman that is doing a internship and i kinda am starting to know what im doing. If i was a example of someone that would like to contribute to a community which is something i really like. I couldnt find it in me to.
This objective of a private AI voice assistant is very interesting and captivating to me, but reading emails isnt very captivating for anyone, or slow messaging. You can have that in specific channels and others for people with diferent expression preferences. I really want this to succeed and not to have to come back 6 months from now and see that it has slowed because of lack of engagement.

**Reply by JarbasAl** [reply]
the email comparison here was to show it is a federated protocol, like you can create an email account in gmail or in hotmail but each can send email for each other, its the same thing for chat, but it is chat! full fledged live chat room you can access in browser or install an app for. 

its fully open source and secure, unlike discord

**Reply by ChanceNCounter** [reply]
I’m currently in 28 Discord servers.

“I am not a kid” wasn’t really the takeaway I was hoping for.

**Reply by VirtualLusamine** [reply]
> the email comparison here was to show it is a federated protocol, like you can create an email account in gmail or in hotmail but each can send email for each other, its the same thing for chat, but it is chat! full fledged live chat room you can access in browser or install an app for. 
> 
> its fully open source and secure, unlike discord

I see that now. Thats a really good explanation thank you. <3

**Comment by ChanceNCounter** [comment, marked as answer]
At any rate, I’m about to repeat something that’s been said many times across the open source community:

IRC is outdated. Matrix won the platform war. I don’t really like it, but it’s where programming is going to happen for the next 10 or 20 years, just like IRC is where it happened for the last 30.

If this is your first exposure to Matrix (or IRC) then welcome aboard! This is an important bit of the open source ecosystem.

Discord, and for that matter Slack and Mattermost, offer some things Matrix doesn’t, but we don’t currently need those things, and if we do, we’re unlikely to choose Discord. I know it feels universal from the inside, but it isn’t, and that’s a very good thing, because gamified communication is terrible for your health.

**Comment by ChanceNCounter** [comment]
Having slept, and woken up, and seen this atop my inbox, I got a certain clarity that I feel might be worth pointing out:

@VirtualLusamine instead of spending two or three minutes looking up what Matrix is, you spent about ten seconds deducing that it is “not an established company.” Strike one. Research is most of what a programmer does.

I told you Matrix is not a company, it’s a service, like email. You took this to mean not that it’s a chat service independent of your host, which is what I meant, but rather than it literally works like email.

Then you wrote hundreds of words trying to explain what Discord is to a game developer, a person who is professionally unable to disengage from Discord. Strike two.

In the course of that essay, you described “the average citizen of New York” as a meth addict.

Hi. My name is Chance. I grew up on a little barrier island just outside New York City limits. Strike three. 18 is *absolutely* a kid, by the way.

**Reply by VirtualLusamine** [reply]
Btw i just wanna point out i was in a situation where i was unable to do proper research as i was at the bank, and was frustrated with something that was affecting me. Its not a excuse i am in the wrong there and ill admit that.
Another think is it seems you took the meth joke a little way to heart. And thats my bad too sorry for not being able to show that as well as i should.
18 is not a kid by the legal defenition is majority of all countries and states. Specially in my country as i have full legal capacity. And alcohol access(alcohol not very relevant but whatever).
The way you called me kid is only relevant on cultures compatible with it. Idk about your culture but in mine im a full adult with all responsibilities and freedoms of such. Socially and Legaly.
And the matrix platform. My main concern isnt really if its a company or not (i didnt express correctly). Its more that i have to provide email and password which is my main worry, though less now. I checked matrix out and it looks interesting so ill check it out in a second.
Again sorry for the dumb way i responded last time.
